### DISCUSSION TITLE
Unable to use bg-blue-400

### DISCUSSION THREAD

**Discussion by nrgbistro** [tailwindlabs/tailwindcss#11483] [Help]
I'm trying to display the true answer in my MC quiz app, but when I attempt to add the className "bg-blue-400" to my component, nothing shows up. I can see bg-yellow-xxx just fine, and even bg-blue-300, bg-blue-500, bg-blue-800, etc. all show up as expected. I've already tried deleting my node_modules and reinstalling tailwindcss but the behavior does not change. I have not modified bg-blue-400 in any tailwind configuration file or index.css. Please let me know how I can debug this issue.

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the `bg-blue-400` class appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.
